scraibe/diarisation.py

```diff
@@ -210,53 +211,64 @@ def load_model(cls,
         Returns:
```

Comment:
Update Doc sting please

Reply:
Fixed this in [55a77b8](https://github.com/JSchmie/ScrAIbe/pull/71/commits/55a77b861cf40e5354e719e511ebe0e6977fe212). Is this what you meant?

Reply:
perfect thx.